src/logcollector/logcollector.c

```diff
@@ -199,7 +200,10 @@ void LogCollectorStart()
             /* Mutexes are not previously initialized under Windows*/
             w_mutex_init(&current->mutex, &win_el_mutex_attr);
 #endif
-
+            free(current->file);
```

Comment:
You can save lines by using `os_free`. Please keep it in mind for future occasions